tests/conftest.py

```diff
@@ -35,11 +35,23 @@ def db():
 
 @pytest.fixture(scope="session")
 def setup_test_environment(db):
+    # Cleaning the existing schema and creating a new one.
+    try:
+        db.engine.execute("drop schema public cascade")
+        db.engine.execute("create schema public")
+    except:
+        db.session.rollback()
+
```

Comment:
Why are we dropping the schema here? I think the flask db downgrade that is added below should work also.